README.md

```diff
@@ -1,12 +1,10 @@
 # Accessible autocomplete
 
----
-
-**We cannot currently respond to issues or pull requests.**
+The accessible autocomplete is a component that helps users choose answers from a list you provide. You can also use it to standardise user input.
```

Comment:
@36degrees Do we need to give an example of how it helps standardise input, or is that something our users will understand?

Reply:
I'm not sure it's clear what we mean by 'standardise user input'. I guess the way the autocomplete does this is to provide _suggestions_ which hopefully users would pick if they matched, but ultimately allow the user to type whatever they want, which is what makes it different from the use case in the first sentence?

Reply:
Should we add something like this?

> For example, when the user of a health service types ‘chest’ - they might see an option that fits their need, but using another term. Similarly, you can use synonyms. For example, to display ‘heart pain’ when the user types ‘chest’.

cc @calvin-lau-sig7

Reply:
How about saying 'make answers more consistent'?
I'm a bit mindful going into an example might push down the rest of the intro, and if so we'd probably want to look for a better or clearer example.

```suggestion
The accessible autocomplete is a component that helps users choose answers from a list you provide. You can also use it to make the answers you get from users more consistent.
```

Reply:
Looks good to me. What do you think, @edwardhorsford and @36degrees?